conf/denas/asr/e2eaiok_denas_asr.conf

```diff
@@ -1,6 +1,6 @@
 # conf for transformer based asr
 model_type: asr
-search_engine: EvolutionarySearchEngine
+search_engine: RandomSearchEngine
```

Comment:
why switch to random search engine?

Reply:
Pruner config uses RandomSearchEngine now, moved this config to Pruner config file

Reply:
was this moved?